packages/formik-inputs/src/components.tsx

```diff
@@ -57,6 +59,10 @@ export const Field: React.FC<FieldProps> = props => (
   <BField {...useField(props)} />
 );
 
+export const FieldSet: React.FC<FieldSetProps> = props => (
```

Comment:
Is this the same as `export { FieldSet, FieldSetProps } from "react-baseline-inputs";`?

Reply:
Ah good call, it definitely is, which makes this superfluous.